### ISSUE TITLE
Define verification key in apt repository description

### ISSUE THREAD

**Issue by ivokub** [open-eid/linux-installer#99]
Instead of installing the key for verifying the repository contents in the global keystore, it should be stored in file system and described in apt repository header. See warning when updating:
```
W: https://installer.id.ee/media/ubuntu/dists/impish/InRelease: Key is stored in legacy trusted.gpg keyring (/etc/apt/trusted.gpg), see the DEPRECATION section in apt-key(8) for details.
```

**Comment by metsma** [comment]
The recent version script already installs to file "/etc/apt/trusted.gpg.d/ria-repository.gpg".
https://github.com/open-eid/linux-installer/blob/master/install-open-eid.sh#L68

**Comment by metsma** [comment]
Ok I see. Your previous comment referred the other deprecated use case
"trusted.gpg keyring (/etc/apt/trusted.gpg) .. apt-key"
Addressed here c28d44330267663f0fc04e40c2f82a63cd67230e

**Comment by ivokub** [comment]
Yes, I initially though that `trusted.gpg` file (which was used by apt-key) and `trusted.gpg.d` directory are equivalent, but it seems these are independent deprecations.
